provider/cldr/Cargo.toml

```diff
@@ -59,6 +59,7 @@ icu_locid_macros = { version = "0.5", path = "../../components/locid/macros" }
 icu_provider_uprops = { version = "0.5", path = "../../provider/uprops", features = ["log"] }
 icu_properties = { version = "0.5", path = "../../components/properties", features = ["provider_serde", "std"] }
 zerovec = { version = "0.6", path = "../../utils/zerovec"}
+elsa = "1.7"
```

Comment:
note: needs approval from dependency cop @zbraniecki

Reply:
taking the emoji react as approval

Reply:
I'm comfortable with deps on datagen.